github/ContentFile.py

```diff
@@ -209,5 +210,6 @@ def _useAttributes(self, attributes):
             self._type = self._makeStringAttribute(attributes["type"])
         if "url" in attributes:  # pragma no branch
             self._url = self._makeStringAttribute(attributes["url"])
+            self._url.value = self._url.value.replace(self._path.value, six.moves.urllib.parse.quote(self._path.value))
```

Comment:
This to me seems like reaching under the covers -- IE, a layering violation. Are we able to quote it when it's still in attributes["url"] ?